lib/ain-rs-exports/src/evm.rs

```diff
@@ -456,11 +456,17 @@ pub fn evm_unsafe_try_construct_block_in_q(
                 total_priority_fees,
                 block_number,
             }) => {
-                let Ok(total_burnt_fees) = u64::try_from(WeiAmount(total_burnt_fees).to_satoshi()) else {
```

Comment:
is there a reason for separating the else into a separate line?

Reply:
sigh.. its `fmt`
https://github.com/DeFiCh/ain/pull/2321/commits/a7ec9f5673089b52f3a203434e96a373cfd9db8f